app/views/groups/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="d-flex flex-fill bg-primary justify-content-between align-items-center text-light p-2">
+  <div><%= link_to "\<", :back, class:"btn text-light btn-sm" %></div>
+  <div class="flex-grow-1 text-center">CATEGORY PAYMENTS</div>
+  <div><small><%= link_to "New", new_payment_path(@group), class: 'text-decoration-none text-white' %></small></div>
+</div>
+<div class="d-flex flex-justify-between p-3 bg-white">
+  <div class="flex-grow-1"><small class="text-muted text-decoration-none"><%= @group.name%> - Total</small></div>
+  <div class="fw-bold"><%=  number_to_currency(@group.payments.sum(:amount))%></div>
+</div>
+<% @group.payments.reverse.each do |payment| %>
+  <div class="m-3 bg-white d-flex gap-3 py-3 px-2">
+    <div class="flex-grow-1"><small class="text-muted text-decoration-none"><%= payment.name %></small></div>
+    <div class="fw-bold"><%= number_to_currency(payment.amount) %></div>
+  </div>
+<% end %>
```

Comment:
- [ ] Please create the big green button at the bottom of the group / category `show` page which shows all the transactions in a given category so that we can add a new transaction item. Currently, you have a link that says "New" in the navigation which does the same thing. Please remove it and create the button instead so that we stick as closely as possible to the design guidelines 👍 
![image](https://user-images.githubusercontent.com/46839548/229303498-a9b2ee33-5c26-4184-bf1e-06c9bda73b24.png)

- [ ] Please remove the links above the navigation. Some of them like "New category" and "New payment" are redundant and are not consistent with the design guidelines
![image](https://user-images.githubusercontent.com/46839548/229303056-232bd4d1-87ab-46a9-a9cc-90f9e2092c58.png)

![image](https://user-images.githubusercontent.com/46839548/229302676-02f63d27-4d84-4034-9d2e-51b189f12ded.png)